<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cRechercheWindow</class>
 <widget class="QWidget" name="cRechercheWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[recherche] - </string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="m_lpTab">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="m_lpBasic">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout" rowstretch="1,0">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="4">
          <widget class="cTextEdit" name="m_lpDescription"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Link:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="cLineEdit" name="m_lpName"/>
         </item>
         <item row="0" column="3">
          <widget class="cLineEdit" name="m_lpLink"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Description:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_lpImages">
      <attribute name="title">
       <string>Images</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QScrollArea" name="m_lpScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>681</width>
            <height>464</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="m_lpLayout"/>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_lpCharacter">
      <attribute name="title">
       <string>Character</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0,0">
           <item>
            <widget class="cComboBox" name="m_lpCharacterList"/>
           </item>
           <item>
            <widget class="QPushButton" name="m_lpCharacterShowDetails">
             <property name="text">
              <string>details...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_lpCaracterAdd">
             <property name="text">
              <string>add...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_lpCharacterRemove">
             <property name="text">
              <string>remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="m_lpCharacterDetails">
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_lpPlace">
      <attribute name="title">
       <string>Place</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0">
           <item>
            <widget class="cComboBox" name="m_lpPlaceList"/>
           </item>
           <item>
            <widget class="QPushButton" name="m_lpPlaceShowDetails">
             <property name="text">
              <string>details...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_lpPlaceAdd">
             <property name="text">
              <string>add...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_lpPlaceRemove">
             <property name="text">
              <string>remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="m_lpPlaceDetails">
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_lpObject">
      <attribute name="title">
       <string>Object</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0">
           <item>
            <widget class="cComboBox" name="m_lpObjectList"/>
           </item>
           <item>
            <widget class="QPushButton" name="m_lpObjectShowDetails">
             <property name="text">
              <string>details...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_lpObjectAdd">
             <property name="text">
              <string>add...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_lpObjectRemove">
             <property name="text">
              <string>remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="m_lpObjectDetails"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ctextedit.h</header>
  </customwidget>
  <customwidget>
   <class>cLineEdit</class>
   <extends>QLineEdit</extends>
   <header>clineedit.h</header>
  </customwidget>
  <customwidget>
   <class>cComboBox</class>
   <extends>QComboBox</extends>
   <header>ccombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_lpName</tabstop>
  <tabstop>m_lpLink</tabstop>
  <tabstop>m_lpDescription</tabstop>
  <tabstop>m_lpTab</tabstop>
  <tabstop>m_lpScrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
